ytmusicapi/parsers/songs.py

```diff
@@ -94,24 +95,72 @@ def parse_song_album(data: JsonDict, index: int) -> JsonDict | None:
     return None if not flex_item else {"name": get_item_text(data, index), "id": browse_id}
 
 
-def parse_song_library_status(item: JsonDict) -> bool:
-    """Returns True if song is in the library"""
-    library_status = nav(item, [TOGGLE_MENU, "defaultIcon", "iconType"], True)
-
-    return library_status == "LIBRARY_SAVED"
-
-
-def parse_song_menu_tokens(item: JsonDict) -> dict[str, str | None]:
-    toggle_menu = item[TOGGLE_MENU]
+def parse_song_menu_data(data: JsonDict) -> JsonDict:
+    """
+    :return: Dictionary with data from the provided song's context menu.
+
+    Example::
+
+        {
+            "inLibrary": true,
+            "feedbackTokens": {
+                "add": "...",
+                "remove": "..."
+            },
+            "pinnedToListenAgain": true,
```

Comment:
This one doesn't show up in the function docstrings in the mixins.

We should also add documentation on how these tokens are expected to be used

I.e. a FAQ entry on "What is the purpose of ``feedbackTokens`` vs ``listenAgainFeedbackTokens``?"